src/Components/Browser.JS/src/Services/UriHelper.ts

```diff
@@ -1,3 +1,5 @@
+import '@dotnet/jsinterop';
```

Comment:
I imagine you are adding this import to force ordering. Could you add a comment explaining?

Reply:
The test started failing saying it could not find a reference to the symbol `DotNet` that we use here. It uses a `ts-jest` to transpile (❓), but I'm not sure why the regular compile so I can see why it's different than the build, but I'm not sure why the regular build doesn't complain about the missing import.